google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -15,12 +15,14 @@
 """
 
 import asyncio
+import aiohttp
 import googleapiclient
 import googleapiclient.discovery
-from googleapiclient.discovery import Resource
 import google.auth
-from google.auth.credentials import Credentials
+from google.oauth2.service_account import Credentials
```

Comment:
Are you required to use service account credentials?

Reply:
No, just strongly encouraged. I'll change it back to `google.auth.credentials`.